packages/jellyfish-api-core/__tests__/category/loan/takeLoan.test.ts

```diff
@@ -175,6 +176,29 @@ describe('Loan takeLoan', () => {
       await tGroup.waitForSync()
     })
 
+    it('should not takeLoan when DFI collateral value less than 50%', async () => {
+      {
+        // reduce DFI value to below 50% of total collateral
+        const now = Math.floor(new Date().getTime() / 1000)
+        await tGroup.get(0).rpc.oracle.setOracleData(oracleId, now, { prices: [{ tokenAmount: '0.4@DFI', currency: 'USD' }, { tokenAmount: '10000@BTC', currency: 'USD' }, { tokenAmount: '2@TSLA', currency: 'USD' }] })
```

Comment:
FYI, you don't need to set price for BTC and TSLA

Reply:
Yea, copy pasted without removed extra data. But no harm and beneficial, in case other tests altered the price.